pkg/snapstore/gcs_snapstore.go

```diff
@@ -68,6 +71,24 @@ func NewGCSSnapStore(config *brtypes.SnapstoreConfig) (*GCSSnapStore, error) {
 			opts = append(opts, option.WithEndpoint(endpoint))
 		}
 	}
+	chunkSuffix := "" // add '@chunk' suffix to chunks folder when using fake gcs storage
```

Comment:
Consider adding a detailed comment here explaining the purpose of chunkSuffix. For instance, you could reference the related GitHub issue or provide reasoning from the fake GCS.
```suggestion
	var chunkSuffix string
```

Reply:
Thanks. Updated this in the latest commit.